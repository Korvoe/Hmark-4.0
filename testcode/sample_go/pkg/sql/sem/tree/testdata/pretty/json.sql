select '[1, "a", true, null, {"c": "blah", "d": []}]':::JSON, '{"\n": ["\""], "\"": "\\"}'::JSONB