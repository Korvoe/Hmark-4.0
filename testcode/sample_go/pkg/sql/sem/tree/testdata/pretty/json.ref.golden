// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
SELECT
	'
		[
			1,
			"a",
			true,
			null,
			{
				"c":
					"blah",
				"d":
					[]
			}
		]
	':::JSONB,
	'
		{
			"\n":
				[
					"\""
				],
			"\"":
				"\\"
		}
	'::JSONB

22:
----------------------
SELECT
	'
		[
			1,
			"a",
			true,
			null,
			{
				"c":
					"blah",
				"d":
					[]
			}
		]
	':::JSONB,
	'
		{
			"\n":
				[
					"\""
				],
			"\"": "\\"
		}
	'::JSONB

23:
-----------------------
SELECT
	'
		[
			1,
			"a",
			true,
			null,
			{
				"c":
					"blah",
				"d": []
			}
		]
	':::JSONB,
	'
		{
			"\n":
				["\""],
			"\"": "\\"
		}
	'::JSONB

25:
-------------------------
SELECT
	'
		[
			1,
			"a",
			true,
			null,
			{
				"c":
					"blah",
				"d": []
			}
		]
	':::JSONB,
	'
		{
			"\n": ["\""],
			"\"": "\\"
		}
	'::JSONB

28:
----------------------------
SELECT
	'
		[
			1,
			"a",
			true,
			null,
			{
				"c": "blah",
				"d": []
			}
		]
	':::JSONB,
	'
		{
			"\n": ["\""],
			"\"": "\\"
		}
	'::JSONB

34:
----------------------------------
SELECT
	'
		[
			1,
			"a",
			true,
			null,
			{"c": "blah", "d": []}
		]
	':::JSONB,
	'
		{"\n": ["\""], "\"": "\\"}
	'::JSONB

39:
---------------------------------------
SELECT
	'
		[
			1,
			"a",
			true,
			null,
			{"c": "blah", "d": []}
		]
	':::JSONB,
	'{"\n": ["\""], "\"": "\\"}'::JSONB

52:
----------------------------------------------------
SELECT
	'
		[1, "a", true, null, {"c": "blah", "d": []}]
	':::JSONB,
	'{"\n": ["\""], "\"": "\\"}'::JSONB

59:
-----------------------------------------------------------
SELECT
	'[1, "a", true, null, {"c": "blah", "d": []}]':::JSONB,
	'{"\n": ["\""], "\"": "\\"}'::JSONB

95:
-----------------------------------------------------------------------------------------------
SELECT
	'[1, "a", true, null, {"c": "blah", "d": []}]':::JSONB, '{"\n": ["\""], "\"": "\\"}'::JSONB

98:
--------------------------------------------------------------------------------------------------
SELECT '[1, "a", true, null, {"c": "blah", "d": []}]':::JSONB, '{"\n": ["\""], "\"": "\\"}'::JSONB


